pkg/operator/controller/status.go

```diff
@@ -94,7 +106,11 @@ func initializeClusterOperator(co *configv1.ClusterOperator, nsName string) {
 			Status: configv1.ConditionUnknown,
 		},
 	}
-	co.Status.RelatedObjects = []configv1.ObjectReference{
+	co.Status.RelatedObjects = defaultRelatedObjects(nsName)
```

Comment:
This is probably not needed because IIRC the status is ignored on create.

Reply:
Makes sense — removed/simplified.